onnxruntime/core/platform/windows/env.cc

```diff
@@ -544,22 +544,21 @@ class WindowsEnv : public Env {
 #endif
     if (!*handle) {
       const auto error_code = GetLastError();
-      LPVOID lpMsgBuf;
+      static const DWORD bufferLength = 64 * 1024;
```

Comment:
why 64*1024?

Reply:
I was referencing this PR : https://github.com/microsoft/onnxruntime/pull/3772/files

What do you suggest?